refarch-backend/src/main/java/de/muenchen/refarch/MicroServiceApplication.java

```diff
@@ -1,25 +1,15 @@
 package de.muenchen.refarch;
 
 import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.annotation.Configuration;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 /**
  * Application class for starting the microservice.
  */
-@Configuration
-@ComponentScan(
-        basePackages = {
-                "org.springframework.data.jpa.convert.threeten",
-                "de.muenchen.refarch"
-        }
-)
 @EntityScan(
```

Comment:
From Baeldung:
@EntityScan annotation is used when entity classes are not placed in the main application package or its sub-packages

We should be able to remove this annotation as well, right?

Reply:
probably yes, i just know that some project use this annotation in their project because they dont have the entity classes inside their main application, thats why i left it in. But in most cases its not needed. i will remove it